src/SchedulerManager/Travely.SchedulerManager.Abstraction/Notifier/INotificationService.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Threading.Tasks;
+
+namespace Travely.SchedulerManager
+{
+    public interface INotificationService
```

Comment:
I think you can split this interface into two separate interfaces: `INotificationService` and `IEmailService` and `Async` suffix like this `NotifyAsync` and `SendEmailAsync`